### PR TITLE
parametrize store and account creation

### PR DESCRIPTION
**Pull request by hpk42**

### CODE REVIEW

File: muacryptcc/plugin.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
+from __future__ import print_function
```
Comment (azul) [inline muacryptcc/plugin.py:1]: why :grey_question:

**Review by azul — APPROVED** [review]
Looks great!

Much more simple and yet more powerful. :+1:

File: muacryptcc/plugin.py (hunk 1 above)
Comment (hpk42) [reply muacryptcc/plugin.py:1]: just a general py2/py3 compat boilerplate.  there should also be unicode_literals but that breaks some of the tests (not sure why yet).

Comment (azul) [reply muacryptcc/plugin.py:1]: @hpk42 but there is no print call in this file.

**Pull request merged**